src/_data/whats-new.yml

```diff
@@ -1,11 +1,63 @@
 title: Whats New on Devdocs
-description: 
+description:
   This page contains recent changes that we think you'd like to know about.
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Sep 14 12:24:21 2020
+updated: Wed Sep 30 08:45:34 2020
 entries:
+- description: Added the Import section to the [Contribution Guide](https://devdocs.magento.com/contributor-guide/contributing.html#import).
+  versions: ''
+  type: Major Update
+  date: September 24, 2020
+  link: https://github.com/magento/devdocs/pull/7886
+  contributor: sidolov
+  profile: https://github.com/sidolov
+- description: Added an optional step to show how to add default values in the [Creating
```

Comment:
Original PR description needs a period.

Reply:
PR 7918 does have a period at the end. I wrote that one. Is something not appearing correctly?

Reply:
Interesting... it should show up in the output in this PR then. Not sure what's up with that.

Reply:
Oh wait, my mistake. The line breaks are throwing me off. No change needed.

Reply:
ahhh..okay. whew!